apps/mobile/src/contexts/MobileAnalyticsProvider.tsx

```diff
@@ -17,18 +17,16 @@ if (token && apiUrl) {
   instance.init(false, {}, apiUrl);
 }
 
-const track: TrackFunction = (eventName, eventProps, userId) => {
+const track: TrackFunction = (eventName, eventProps) => {
   try {
-    instance?.track(eventName, { userId, platform: 'mobile', ...eventProps });
+    instance?.track(eventName, eventProps);
```

Comment:
1) `userId` tracking is unnecessary as the `identify` function will attach a `User ID` key to all track events out of the box. from looking at the data, `User ID` is much more reliable than our own `userId`

2) remove `platform: mobile` tracking as a more reliable piece is the `Mixpanel Library` key, which again is attached out of the box